internal/pagination/pagination.go

```diff
@@ -56,19 +56,14 @@ type ListFilterFunc[T boundary.Resource] func(ctx context.Context, item T) (bool
 // If prevPageLastItem is empty, it returns a slice of T starting with the least recently updated.
 type ListItemsFunc[T boundary.Resource] func(ctx context.Context, prevPageLastItem T, limit int) ([]T, error)
 
-// ListRefreshItemsFunc returns a slice of T that have been updated since prevPageLastItem.
-// If prevPageLastItem is empty, it returns a slice of T that have been updated since the
-// item in the refresh token.
-type ListRefreshItemsFunc[T boundary.Resource] func(ctx context.Context, tok *refreshtoken.Token, prevPageLastItem T, limit int) ([]T, error)
```

Comment:
This function was no longer necessary, and it simplifies both the service functions and the pagination logic not to use it.